src/executors/python-default.yml

```
@@ -0,0 +1,23 @@
+description: Common cimg/node image executor
+parameters:
+  repo:
+    type: string
```

I don't think we need a repo param but if we do then we should default to circle ci's default `cimg/pthon`
```suggestion
  repo:
    type: string
    default: cimg/python
```

---

I can make these suggestions as well as update [source-inventory](https://github.com/GoodwayGroup/source-inventory/blob/3ffd9f87621a7f12b8bdbadd3ff9d152b6bec519/.circleci/config.yml#L11) which is the only repo that overrides the `executor-repo` to use the current python image.